app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    @review = current_user.reviews.build(review_params)
+    @review.video = @video
```

Comment:
You could rather pass it to the params above and mass assign this rather than creating a new line.

` @review = current_user.reviews.build(review_params.merge!(video: @video))`